wandb/sdk/launch/sweeps/scheduler_sweep.py

```diff
@@ -144,7 +149,6 @@ def _run(self) -> None:
         )
         _ = self._add_to_launch_queue(
             run_id=run.id,
-            entry_point=["python3", run.program] if run.program else None,
```

Comment:
we could also just pipe in the sweep config entry point here, and then handle disambiguating between them when creating the job params. Currently this would override though....

Reply:
I think it could end up being important to allow users to customize the entrypoint when creating a sweep from a job. Today they are stuck with the entrypoint from the original job, and I could see that being annoying. The user would have to create a new job with the exact entrypoint they want just to be able to run a sweep.

I would say using the job entrypoint by default is reasonable but maybe we should still allow users to override that with the "command" section from the sweep config. https://docs.wandb.ai/guides/sweeps/define-sweep-configuration#command

Maybe we could also add a macro to the command config for the job entrypoint like ${job_entrypoint} or something like that.

Reply:
Okay I hear you, but doesn't that feel like something that could be solved (at least in the UI) by displaying the current job launch spec, and then making it editable? Not exactly sure how to do that in the SDK, but you could always just include a launch-spec when creating a sweep with different override args...